resources/views/page_builder/brand_overview.blade.php

```diff
@@ -0,0 +1,52 @@
+@php
+    $brands = \Statamic\Facades\Entry::query()
+        ->where('collection', 'brands')
```

Comment:
This query is missing the site check.
`->where('site', \Statamic\Facades\Site::current()->handle())` should do the trick!